### PR TITLE
Reduce yaml files for ha_textmode

### PR DESCRIPTION
**Pull request by lemon-suse**
- Related ticket: https://progress.opensuse.org/issues/156031
- Related MR: https://gitlab.suse.de/qe-yam/openqa-job-groups/-/merge_requests/245
- Needles: N/A
- Verification run: http://openqa.suse.de/tests/overview?version=15-SP6&build=lemon-suse%2Fos-autoinst-distri-opensuse%23reduce-yaml-ha-textmode&distri=sle

### CODE REVIEW

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
```
Comment (jknphy) [inline schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:1]: I don't think this is particulary true only from `ha` it is a `sles` feature, you can drop that from the name:
`default_s390x_kvm_textmode.ymal`

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+# Default ordered sequence of steps to be optionally overwritten for s390x kvm
```
Comment (jknphy) [inline schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:2]: s/kvm/kvm textmode/

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml (hunk 1 above)
Comment (lemon-suse) [reply schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:1]: The schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml already existed, the name of default_s390x_kvm_ha_textmode.yaml just means specific for ha_textmode test suite.

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+license_agreement:
+  - installation/licensing/accept_license
+registration:
+  - installation/registration/register_via_scc
+extension_module_selection:
+  - installation/module_registration/skip_module_registration
+add_on_product:
+  - installation/add_on_product/skip_install_addons
+system_role:
+  - installation/system_role/accept_selected_role_text_mode
```
Comment (jknphy) [inline schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:17]: I cannot access OSD at the moment but wondering if we need to overwrite then in schedule/yam/test_cases/ha_textmode.yaml the system role or we can drop that in that file.

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml (hunk 1 above)
Comment (jknphy) [reply schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:1]: then as @hjluo mentioned, we don't need to duplicate files, you can reuse it, those extra preparation test modules can be added to ha. You need verification for the test suite currently using it.

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml (hunk 3 above)
Comment (lemon-suse) [reply schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:17]: I will run a test to check whether there is blocker to run accept_selected_role_text_mode instead, if no I think we can remove that overwrite in ha_textmode.yaml.

Comment (jknphy) [reply schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:17]: osd is back, we can't, the ha role is preselected:
https://openqa.suse.de/tests/14611192#step/select_role_text_mode/1

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml (hunk 1 above)
Comment (lemon-suse) [reply schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml:1]: Ok, reuse the schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml

File: schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml (hunk 2 above)
Comment (lemon-suse) [reply schedule/yast/sle/flows/default_s390x_kvm_ha_textmode.yaml]: Don't create new yaml file, reuse the default_s390x_kvm_textmode.yaml.

**Pull request merged**